host-configs/rzansel-blueos_3_ppc64le_ib_p9-clang@8.0.1_nvcc_xlf.cmake

```diff
@@ -100,7 +100,7 @@ set(BLT_EXE_LINKER_FLAGS "-Wl,-rpath,/usr/tce/packages/xl/xl-2019.12.23/lib" CAC
 
 set(ENABLE_CUDA ON CACHE BOOL "")
 
-set(CUDA_TOOLKIT_ROOT_DIR "/usr/tce/packages/cuda/cuda-10.1.168" CACHE PATH "")
+set(CUDA_TOOLKIT_ROOT_DIR "/usr/tce/packages/cuda/cuda-10.1.243" CACHE PATH "")
```

Comment:
It may be good to set `AXOM_USE_CUB=ON` on this host-config, so that we have one host-config that tests that functionality as well.

Reply:
im not rebuilding host-configs again... feel free to do that after this is merged.

Reply:
Sure, we can do later -- it doesn't require a rebuild, we just set it in the host-config and update the axom's spack package when it generates the host-config.